bin/cli.js

```diff
@@ -420,9 +422,14 @@ else if (scripts.length === 0) {
   runStream(process.stdin, process.stdout, mode, parenthesis);
 }
 else {
-  //work through the queue of scripts
-  scripts.forEach(function (arg) {
-    // run a script file
-    runStream(fs.createReadStream(arg), process.stdout, mode, parenthesis);
-  });
+  fs.stat(scripts[0], (e, f) => {
+    if (e)
```

Comment:
I personally prefer explicit if/else and always use code blocks like `if (...) {...} else {....}` for more robust code, but it's not a big deal.